sklearnex/dispatcher.py

```diff
@@ -32,6 +32,10 @@
     from .svm import NuSVR as NuSVR_sklearnex
     from .svm import NuSVC as NuSVC_sklearnex
 
+    from .neighbors import KNeighborsClassifier as KNeighborsClassifier_sklearnex
+    # from .neighbors import KNeighborsRegressor as KNeighborsRegressor_sklearnex

```

Comment:
remove?

Reply:
we discussed to left it as a reminder for the following patching expanding. should i remove it?

Reply:
Ok, let it be here